rotkehlchen/inquirer.py

```diff
@@ -722,14 +718,14 @@ def _find_usd_price(
         )
         return price, oracle, used_main_currency
 
-    def find_uniswap_v2_lp_price(
+    def find_lp_price_from_pool_contract(
             self,
             token: EvmToken,
     ) -> Optional[Price]:
-        # BAD BAD BAD. TODO: Need to rethinking placement of modules here
-        from rotkehlchen.chain.ethereum.modules.uniswap.utils import find_uniswap_v2_lp_price  # isort:skip  # noqa: E501  # pylint: disable=import-outside-toplevel
-        return find_uniswap_v2_lp_price(
-            ethereum=self.get_ethereum_manager().node_inquirer,
+        """Calculates the price for an LP token the contract of which is also the contract of
```

Comment:
mention uniswaplike here, but rest of description sounds okay
```suggestion
        """Calculates the price for a uniswaplike LP token the contract of which is also the contract of
```